jalib/jassert.cpp

```diff
@@ -46,6 +43,9 @@ int jassert_quiet = 0;
 static int theLogFileFd = -1;
 static int errConsoleFd = -1;
 
+static jalib::string& tmpDir() {static jalib::string s;return s;};
+static jalib::string& uniquePidStr() {static jalib::string s;return s;};
```

Comment:
This is another good point.  The comment that I suggested was just
a first draft.  Probably @kara0 and you can improve on it.  Maybe,
we could make the comment shorter by pointing to:
https://isocpp.org/wiki/faq/ctors#static-init-order,
and https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use
I'm good with however you choose to word the comment.